src/invidious/routing.cr

```diff
@@ -1,15 +1,13 @@
 module Invidious::Routing
   macro get(path, controller, method = :handle)
     get {{ path }} do |env|
-      controller_instance = {{ controller }}.new
-      controller_instance.{{ method.id }}(env)
+      {{ controller }}.{{ method.id }}(env)
     end
   end
 
   macro post(path, controller, method = :handle)
     post {{ path }} do |env|
-      controller_instance = {{ controller }}.new
-      controller_instance.{{ method.id }}(env)
+      {{ controller }}.{{ method.id }}(env)
```

Comment:
While you're at it, can you add macros for `delete` and `options`? Those will be required in the future ;)